smtp panic EOF

smtp conf：
  # Skip TLS certificate verify when connecting to SMTP server
  no-verify = false
  # Close idle connections after timeout
  idle-timeout = "2s"

panic: EOF

goroutine 8 [running]:
panic(0xc62140, 0xc8200120d0)
	/usr/local/go/src/runtime/panic.go:481 +0x3e6
github.com/influxdata/kapacitor/services/smtp.(*Service).runMailer(0xc820198be0)
	/root/go/src/github.com/influxdata/kapacitor/services/smtp/service.go:91 +0x7ad
created by github.com/influxdata/kapacitor/services/smtp.(*Service).Open
	/root/go/src/github.com/influxdata/kapacitor/services/smtp/service.go:36 +0x238

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -31,6 +31,7 @@ kapacitor replay-live query -task cpu_alert -query 'SELECT usage_idle FROM teleg
 - [#521](https://github.com/influxdata/kapacitor/issues/521): EvalNode now honors groups.
 - [#561](https://github.com/influxdata/kapacitor/issues/561): Fixes bug when lambda expressions would return error about types with nested binary expressions.
 - [#555](https://github.com/influxdata/kapacitor/issues/555): Fixes bug where "time" functions didn't work in lambda expressions.
+- [#570](https://github.com/influxdata/kapacitor/issues/570): Removes panic in SMTP service on failed close connection.
 
 ## v0.13.1 [2016-05-13]
 

--- a/services/smtp/service.go
+++ b/services/smtp/service.go
@@ -88,7 +88,7 @@ func (s *Service) runMailer() {
 		case <-time.After(time.Duration(s.c.IdleTimeout)):
 			if open {
 				if err := conn.Close(); err != nil {
-					panic(err)
+					s.logger.Println("E! error closing connection to SMTP server:", err)
 				}
 				open = false
 			}

